Assets/Scripts/Models/Buildable/Furniture.cs

```diff
@@ -756,8 +788,25 @@ public Inventory[] AcceptsForStorage()
     /// <summary>
     /// Deconstructs the furniture.
     /// </summary>
-    public void Deconstruct()
+    public void SetDeconstructJob()
     {
+        Jobs.CancelAll();
```

Comment:
Okay, I'm about to submit an update that I think resolves this. Let me know what you think!